apps/src/code-studio/components/BubbleChoice.jsx

```diff
@@ -69,7 +70,9 @@ export default class BubbleChoice extends React.Component {
               />
             )}
             <div style={styles.column}>
-              <span style={styles.title}>{sublevel.title}</span>
+              <a href={sublevel.url + location.search} style={styles.title}>
```

Comment:
UX question: should I also go to the level if I click on the image?

Reply:
most likely, but i have a UX meeting coming up with Amanda and Elizabeth, so i'm holding off on polishing this page too much right now